package.json

```diff
@@ -5,6 +5,7 @@
   "main": "lib/cosmos.js",
   "scripts": {
     "test": "jest",
+    "lint": "eslint src/* --fix",
```

Comment:
i don't want to add this to your plate right now, but we should consider making a template repo for lunie projects where the package.json scripts, dev dependencies, linting and testing configurations are the same across all our projects.

Reply:
Already looked into that but I didn't want to work on this right now :)